localisation/000_Other_title_names.csv

```diff
@@ -367,4 +367,14 @@ b_menethil_adj;Lordain;;;;;;;;;;;;;x
 b_legation_quarter;Stanford;;;;;;;;;;;;;x
 b_legation_quarter_adj;Stanford;;;;;;;;;;;;;x
 b_great_cathedral;Anderton;;;;;;;;;;;;;x
-b_great_cathedral_adj;Anderton;;;;;;;;;;;;;x
\ No newline at end of file
+b_great_cathedral_adj;Anderton;;;;;;;;;;;;;x
+d_queldanas_isle;Quel'Danas Isle;Quel'Danas Isle;Quel'Danas Isle;;Quel'Danas Isle;;;;;;;;;x
+d_queldanas_isle_adj;Quel'Danasean;;Quel'Danassisch ;;;;;;;;;;;x
+e_quelthalas;Quel'Thalas;Quel'Thalas;Quel'Thalas;;Quel'Thalas;;;;;;;;;x
+e_quelthalas_adj;Quel'Thalasian;;quel'thalaser ;;;;;;;;;;;x
```

Comment:
**_Thalassian_** sounds more legit here.
https://wow.gamepedia.com/Thalassian

Reply:
agree and done